web/nouns/models.py

```diff
@@ -209,12 +209,16 @@ def relation_type_label(self):
 
 
 class Channel(models.Model):
+    class Meta:
+        ordering = ('order',)
+
     title = models.CharField(max_length=255)
     slug = models.CharField(max_length=255)
     nouns = models.ManyToManyField('Noun', null=True, blank=True)
-    order = models.IntegerField()
-    language = models.CharField(max_length=255, blank=True, null=True)
-    is_featured = models.BooleanField(max_length=255, default=False)
+    order = models.IntegerField(default=0)
+    language = models.CharField(max_length=255, choices=settings.LANGUAGES, default=get_language)
+    # Translators: this is a Channel's is_featured field
+    is_featured = models.BooleanField(max_length=255, default=False, help_text=_('If set this channel will be shown on the main page'))
```

Comment:
This change needs Django migrations. If any model property changed (DB changes or not), User should generate Django Migration.

Can you add Django migrations about it ?

Reply:
I've added migrations for nouns. There were some other missing premises migrations though (e8fd225d0f35f680d8a9) . I've added them as a #366 PR and also included here. If they look ok, just merging this PR will solve the other as well.

Reply:
BTW: It's weird that non-db changes are tracked in migrations as well. It has been resolved in Django 1.9.1 apparently, here is a hotfix to disable it: https://stackoverflow.com/questions/26503826/why-does-django-generate-migrations-for-help-text-and-verbose-name-changes